MoltenVK/MoltenVK/Commands/MVKCommandEncoderState.mm

```diff
@@ -333,33 +333,16 @@
 }
 
 void MVKRenderingCommandEncoderState::setDepthBias(const VkPipelineRasterizationStateCreateInfo& vkRasterInfo) {
-	bool isDynamic = false;
-
-	bool mtlDepthBiasEnable = static_cast<bool>(vkRasterInfo.depthBiasEnable);
-	setMTLContent(DepthBiasEnable);
-
-	MVKDepthBias mtlDepthBias = {
-		.depthBiasConstantFactor = vkRasterInfo.depthBiasConstantFactor,
-		.depthBiasSlopeFactor = vkRasterInfo.depthBiasSlopeFactor,
-		.depthBiasClamp = vkRasterInfo.depthBiasClamp
-	};
-	setMTLContent(DepthBias);
+	setDepthBiasEnable(vkRasterInfo.depthBiasEnable, false);
+	setDepthBias( { vkRasterInfo.depthBiasConstantFactor, vkRasterInfo.depthBiasClamp, vkRasterInfo.depthBiasSlopeFactor } , false);
 }
 
-void MVKRenderingCommandEncoderState::setDepthBias(float depthBiasConstantFactor,
-													 float depthBiasSlopeFactor,
-													 float depthBiasClamp) {
-	bool isDynamic = true;
-	MVKDepthBias mtlDepthBias = {
-		.depthBiasConstantFactor = depthBiasConstantFactor,
-		.depthBiasSlopeFactor = depthBiasSlopeFactor,
-		.depthBiasClamp = depthBiasClamp
-	};
+void MVKRenderingCommandEncoderState::setDepthBias(MVKDepthBias depthBias, bool isDynamic) {
+	MVKDepthBias mtlDepthBias = depthBias;
```

Comment:
I wonder if we are now at the point where we could refactor `setMTLContent()` not to need this copy.